components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -56,14 +58,15 @@ class AddonsManagerAdapter(
     private val addonsManagerDelegate: AddonsManagerAdapterDelegate,
     addons: List<Addon>,
     private val style: Style? = null
-) : RecyclerView.Adapter<CustomViewHolder>() {
+) : ListAdapter<Any, CustomViewHolder>(DifferCallback) {
```

Comment:
ListAdapter<**Any**, CustomViewHolder>(DifferCallback) instead of `Addon` because of the sections, right? I remember doing something similar in the past without needing to declare generic type using DiffUtil 🤔

Reply:
Exactly, as we have three(Sections,add-ons and unsupported  add-ons section) types that can be pass but don't have any relations between each other.